build/package/Dockerfile.java-toolset

```diff
@@ -0,0 +1,17 @@
+FROM adoptopenjdk/openjdk11:ubi
```

Comment:
I think I'd prefer [ubi8/openjdk-11](https://catalog.redhat.com/software/containers/ubi8/openjdk-11/5dd6a4b45a13461646f677f4) to stick with RedHat-provided images. Since Java 17 is coming out soon, we can switch to that once it is available from RedHat in order to profit from all the latest developments.

Reply:
I would change it but I get this error:
`
Step 1/8 : FROM ubi8/openjdk-11:1.10
pull access denied for ubi8/openjdk-11, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
`

Reply:
You need to pull from RedHat's registry: `registry.redhat.io/ubi8/openjdk`